windows_docker_resources/install_ros2_iron.json

```diff
@@ -0,0 +1,24 @@
+{
+  "ros2_windows": {
+    "download_sources": "false",
+    "vs_version": "buildtools",
+    "qt5": {
+      "installation_method": "offline"
+    },
+    "ros2_ws": "C:/ci",
+    "ros_distro": "rolling",
```

Comment:
Should this be `iron`?

Reply:
yes it should. I copied from rolling and forgot to update the contents.